File: PYTHON/Program/Komputer.py
from Cpu import Cpu
from Ram import Ram
from HybridStorage import HybridStorage

class Komputer:
  def __init__(self, nama="", cpu=None, ramList=None, hybridStorage=None):
    self.nama = nama
    self.cpu = cpu if cpu is not None else Cpu()
    self.ramList = ramList if ramList is not None else []
    self.hybridStorage = hybridStorage if hybridStorage is not None else HybridStorage()

  def setNama(self, nama):
    self.nama = nama

  def setCpu(self, cpu):
    self.cpu = cpu

  def setRam(self, ramList):
    self.ramList = ramList

  def setHybridStorage(self, hybridStorage):
    self.hybridStorage = hybridStorage

  def addRam(self, ram):
    self.ramList.append(ram)

  def getNama(self):
    return self.nama

  def getCpu(self):
    return self.cpu

  def getRamList(self):
    return self.ramList

  def getHybridStorage(self):
    return self.hybridStorage

File: JAVA/Program/Komputer.java
import java.util.ArrayList;
import java.util.List;

public class Komputer {
  private String nama;
  private Cpu cpu;
  private List<Ram> ramList;
  private HybridStorage hybridStorage;

  public Komputer() {
    this.ramList = new ArrayList<>();
  }

  public Komputer(String nama, Cpu cpu, List<Ram> ramList, HybridStorage hybridStorage) {
    this.nama = nama;
    this.cpu = cpu;
    this.ramList = ramList;
    this.hybridStorage = hybridStorage;
  }

  public void setNama(String nama) {
    this.nama = nama;
  }

  public void setCpu(Cpu cpu) {
    this.cpu = cpu;
  }

  public void setRam(List<Ram> ramList) {
    this.ramList = ramList;
  }

  public void setHybridStorage(HybridStorage hybridStorage) {
    this.hybridStorage = hybridStorage;
  }

  public void addRam(Ram ram) {
    this.ramList.add(ram);
  }

  public String getNama() {
    return this.nama;
  }

  public Cpu getCpu() {
    return this.cpu;
  }

  public List<Ram> getRamList() {
    return this.ramList;
  }

  public HybridStorage getHybridStorage() {
    return this.hybridStorage;
  }
}
